draft-ietf-tls-dtls-rrc.md

```diff
@@ -172,18 +172,26 @@ layer specific address validation mechanism can be triggered instead (e.g., CoAP
 
 # Attacker Model {#attacker}
 
-We define two classes of attackers, off-path and on-path, with increasing
-capabilities (see {{fig-attacker-capabilities}}) partly following terminology
+Attacks are divided into passive and active attacks. Passive attackers have the ability
+to read packets from the network, while active attackers also have the ability to write
+packets into the network. However, a passive attack could involve an attacker with the
+ability to cause a routing change or other modification in the path taken by packets that
+comprise a connection.
```

Comment:
Attackers are not assumed to have control over L3 routing.  Instead, they can observe packets on one network segment and make (modified) copies on another segment and win the race to the destination due to, e.g., better SLA.

Reply:
This is what the original L179-182 try to convey.

Reply:
I copied the text from of Section 21.1 of QUIC.

Reply:
https://www.rfc-editor.org/rfc/rfc9000#name-overview-of-security-proper

Reply:
Ah ok.  Reading that bit from QUIC makes it seem the attacker is more powerful than it needs to be though.   In my understanding, it is actually a very specific L4 routing manipulation that is assumed, rather than full-blown control over L3 paths.

Reply:
Obviously, if the attacker is a router that can inject "better routes" through itself, it can be classified as an off-path attacker in our (i.e., QUIC) terminology.  But, it's not necessary to be that powerful.